cs50/2017/fall/recover/check50/__init__.py

```diff
@@ -1,58 +1,35 @@
 from check50 import *
+import os
+import re
 
 class Recover(Checks):
-    hashes = [
```

Comment:
What's the reasoning for removing these? It seems to just complicate the check and add parsing overhead to every check, no?

Reply:
To make it easier to update the check for new sets of JPEGs.

Reply:
Yeah, the additional parsing doesn't feel like a big deal. Much easier to update a config file each term than manually format 50 new strings in the code itself.